aopy/precondition.py

```diff
@@ -45,7 +45,7 @@ def butterworth_filter_data(data,fs, cutoff_freq= None, bands= None,  order= Non
     Apply a digital butterworth filter forward and backward to a timeseries signal.
 
     Args:
-        data (array): neural data (n_channels x n_samples)
+        data (array): neural data (n_samples x n_channels)
```

Comment:
Can you move the size of the array into the parentheses? See the docstring writing guide https://analyze.readthedocs.io/en/latest/documentation-guidelines.html